### DISCUSSION TITLE
alist+aria2，下载夸克网盘文件时内存占用飙升

### DISCUSSION THREAD

**Discussion by coderains** [AlistGo/alist#7624] [Q&A]
群晖docker安装alist+aria2，下载夸克网盘文件时内存占用飙升，下载别的网盘文件没事

**Comment by nemosea** [comment]
补充下我遇到的问题。aria2默认配置64线程，块大小64M,下载速度30+M，大概十几G之后就会导致系统卡死，主机为N1，2G内存。
限制容器内存为512M，会很快触发alist被oom，导致aria2下载失败。
临时方案，限制aria2最大8线程现在，alist内存占用从40+M 到 300+M的一个变动，文件大小56G，下载速度30+M.
@xhofe

**Reply by pongfcnkl** [reply]
> 补充下我遇到的问题。aria2默认配置64线程，块大小64M,下载速度30+M，大概十几G之后就会导致系统卡死，主机为N1，2G内存。 限制容器内存为512M，会很快触发alist被oom，导致aria2下载失败。 临时方案，限制aria2最大8线程现在，alist内存占用从40+M 到 300+M的一个变动，文件大小56G，下载速度30+M. @xhofe

更新一下beta，还有你线程太多了，而且夸克只能2线程

**Reply by ChanLicher** [reply]
我试着更新到Beta了，然后也限制最大线程到8，同时限制Alist的内存占用上限为6G，然后我测试下载夸克的大型文件就是缓慢地占满6G上限，这感觉没有彻底解决问题。

**Comment by pongfcnkl** [comment]
用rclone下
